examples/2d/mesh2d_manual.rs

```diff
@@ -70,8 +70,8 @@ fn star(
     // Set the position attribute
     star.insert_attribute(Mesh::ATTRIBUTE_POSITION, v_pos);
     // And a RGB color attribute as well
-    let mut v_color = vec![[0.0, 0.0, 0.0, 1.0]];
-    v_color.extend_from_slice(&[[1.0, 1.0, 0.0, 1.0]; 10]);
+    let mut v_color: Vec<u32> = vec![bytemuck::cast([0_u8, 0_u8, 0_u8, 255_u8])];
```

Comment:
This change shouldn't be in this PR; did something go wrong? Do you need to rebase?

Reply:
Yes, You are right. It should be fixed now I guess.